### PR TITLE
JS2 week1 homework - added

### PR DESCRIPTION
**Pull request by vitalreddym**

### CODE REVIEW

File: javascript/javascript2/week1/animal.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const button = document.querySelector("#myButton");
+const outputPara = document.createElement("p");
+
+button.addEventListener("click", function () {
+    let randomNumber = adjectives[Math.floor(Math.random() * adjectives.length)] + " " + spiritAnimals[Math.floor(Math.random() * spiritAnimals.length)];
+    const user = document.querySelector("#userInput").value;
+    if (user === "" || user === "null") {
+        alert("Enter your name to generate your spirit animal");
+    } else {
+        outputPara.innerText = "Your spirit animal is: " + user + " - " + randomNumber;
```
Comment (ElhamDargi) [inline javascript/javascript2/week1/animal.js:40]: Works nicely 🧚‍♀️

You can also write this line like this.
 outputPara.innerText = `Your spirit animal is: ${user}- ${randomNumber}`;

File: javascript/javascript2/week1/main.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    let list = document.createElement("li");
+    list.appendChild(productTitle);
+    list.appendChild(productPrice);
+    list.appendChild(productRating);
+    ul.appendChild(list);
+}
+}
+
+renderProducts(products);
```
Comment (ElhamDargi) [inline javascript/javascript2/week1/main.js:27]: Excellent, well done!

File: javascript/javascript2/week1/numbers.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+console.log( "The shortest danishword is " + shortWord);
+
+
+
+// Find & Count Danish letters
+
+// total number of danish letters
+const danishLetters = ["æ", "å", "ø"];
+
+function findDanishLetters (danishString) {
```
Comment (ElhamDargi) [inline javascript/javascript2/week1/numbers.js:24]: So good, Great solution.

File: javascript/javascript2/week1/animal.js (hunk 1 above)
Comment (vitalreddym) [reply javascript/javascript2/week1/animal.js:40]: ok, sure I will keep in mind

**Pull request merged**